internal/configs/configs.go

```diff
@@ -223,29 +273,67 @@ func backupFile(source string) {
 
 func makeBackupDir(dest string) {
 	// If a backup directory does not exist
-	if !fileio.FileExist("backup/") {
+	exists, err := fileio.FileExist("backup/")
+	if err != nil {
+		// If we received an error that is not ErrNotExist
+		common.ErrorCheck(err, "Error checking for backup/ folder")
+		return // note: unreachable
+	}
+
+	if !exists {
 		// Write to the logger
 		logger.Printf("Backup directory does not exist!\nCreating backup directory for first run backup")
 	}
 
 	// Make the empty directories
-	err := os.MkdirAll(fmt.Sprintf("backup/%s", dest), os.ModePerm)
-	errorcheck.ErrorCheck(err, "Error making backup/ folder")
+	if err = os.MkdirAll(fmt.Sprintf("backup/%s", dest), os.ModePerm); errors.Is(err, os.ErrExist) {
+		// ignore if the directory already exists
+		err = nil
+	}
+	// will return without incident if there's no error
+	common.ErrorCheck(err, "Error making backup/ folder")
 }
 
-// Copy a file to the system, make sure you have run command.Elevate() recently
-func CopyToSystem(conffile, sysfile string) string {
+// CopyToSystem copies a file to the system.
+func CopyToSystem(isRoot bool, conffile, sysfile string) {
 	// Since we should be elevated with our sudo token we will copy with cp
 	// (using built in functions will not work as we are running as the normal user)
-	output, _ := command.Run("sudo", "cp", "-v", conffile, sysfile)
 
-	// Clean the output
-	clean_re := regexp.MustCompile(`\n`)
-	clean_output := clean_re.ReplaceAllString(output[0], "")
+	// ExecAndLogSudo will write to the logger, so just print here
+	fmt.Printf("Copying: %s to %s\n", conffile, sysfile)
+
+	if isRoot {
+		logger.Printf("Copying %s to %s\n", conffile, sysfile)
+		fmt.Printf("Copying %s to %s\n", conffile, sysfile)
+		fDat, err := os.ReadFile(conffile)
+		common.ErrorCheck(err, fmt.Sprintf("Failed to read %s", conffile))
+		err = os.WriteFile(sysfile, fDat, 0644)
+		common.ErrorCheck(err, fmt.Sprintf("Failed to write %s", sysfile))
+		logger.Printf("Copied %s to %s\n", conffile, sysfile)
+		return
+	}
+
+	if !filepath.IsAbs(conffile) {
+		conffile, _ = filepath.Abs(conffile)
+	}
+
+	conffile = strings.ReplaceAll(conffile, " ", "\\ ")
+	cmd := fmt.Sprintf("cp -v %s %s", conffile, sysfile)
+
+	err := command.ExecAndLogSudo(isRoot, false, cmd)
```

Comment:
I tested everything but i had to patch this into
```go
err := command.ExecAndLogSudo(isRoot, false, "cp", "-v", conffile, sysfile)
```

in order for everything to work as it got missed when we changed how ExecAndLogSudo worked. i will just patch that after merge